# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: ipc/glue/BackgroundParentImpl.h
# Commit: 989cbb08b87e
# Full Hash: 989cbb08b87e92e4192fc152d0b3e352240ab330
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2023-01-20 21:21:03
# Regressor Bug: 1351231
# File Overlap Count: 2
# Description:
#   Bug 1351231 - PFetch protocol declaration and implementation. r=dom-worker-reviewers,jesup
#   
#   Depends on D138813
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D142436
# ==============================================================================

diff -r d77b78a71a7e -r 989cbb08b87e ipc/glue/BackgroundParentImpl.h
--- a/ipc/glue/BackgroundParentImpl.h	Fri Jan 20 09:09:17 2023 +0000
+++ b/ipc/glue/BackgroundParentImpl.h	Fri Jan 20 09:09:17 2023 +0000
@@ -415,6 +415,8 @@
 
   already_AddRefed<PLockManagerParent> AllocPLockManagerParent(
       const ContentPrincipalInfo& aPrincipalInfo, const nsID& aClientId) final;
+
+  already_AddRefed<PFetchParent> AllocPFetchParent() override;
 };
 
 }  // namespace mozilla::ipc